integrationtest/mapper/pojo-base/src/test/java/org/hibernate/search/integrationtest/mapper/pojo/mapping/definition/IndexedEmbeddedBaseIT.java

```diff
@@ -1617,7 +1822,480 @@ class EntityC {
+				}
+			}
+		}
+
+		backendMock.expectSchema( INDEX_NAME, b -> b
+				.field( "indexedEntityString", String.class )
+				.objectField( "included", b2 -> b2
+						.field( "includedString", String.class )
+						.objectField( "subIncluded", b3 -> b3
+								.field( "subIncludedStringB", String.class )
+								.field( "subIncludedStringC", String.class )
+						)
+				)
+		);
+
+		SearchMapping mapping = setupHelper.start()
+				.withAnnotatedEntityTypes( Model.IndexedEntity.class )
+				.setup();
+
+		backendMock.verifyExpectationsMet();
+
+		Model model = new Model();
+
+		doTestEmbeddedRuntime(
+				mapping,
+				id -> model.new IndexedEntity(
+						id, "a",
+						model.new Included( "b", model.new SubIncluded(
+								"c", "d", "e" )
+						)
+				),
+				document -> document.field( "indexedEntityString", "a" )
+						.objectField( "included", b2 -> b2
+								.field( "includedString", "b" )
+								.objectField( "subIncluded", b3 -> b3
+										.field( "subIncludedStringB", "d" )
+										.field( "subIncludedStringC", "e" )
+								)
+						)
+		);
+	}
+
+	@Test
+	public void parentExcludeChildExcludeSameProperty() {
+		class Model {
+			@Indexed(index = INDEX_NAME)
+			class IndexedEntity {
+				@DocumentId
+				Integer id;
+				@KeywordField
+				String indexedEntityString;
+
+				@IndexedEmbedded(excludePaths = "subIncluded.subIncludedStringA")
+				Included included;
+
+				public IndexedEntity(Integer id, String indexedEntityString, Included included) {
+					this.id = id;
+					this.indexedEntityString = indexedEntityString;
+					this.included = included;
+				}
+			}
+
+			class Included {
+				@KeywordField
+				String includedString;
+
+				@IndexedEmbedded(excludePaths = "subIncludedStringA")
+				SubIncluded subIncluded;
+
+				public Included(String includedString, SubIncluded subIncluded) {
+					this.includedString = includedString;
+					this.subIncluded = subIncluded;
+				}
+			}
+
+			class SubIncluded {
+				@KeywordField
+				String subIncludedStringA;
+				@KeywordField
+				String subIncludedStringB;
+				@KeywordField
+				String subIncludedStringC;
+
+				public SubIncluded(String subIncludedStringA, String subIncludedStringB, String subIncludedStringC) {
+					this.subIncludedStringA = subIncludedStringA;
+					this.subIncludedStringB = subIncludedStringB;
+					this.subIncludedStringC = subIncludedStringC;
+				}
+			}
+		}
+
+		assertThatThrownBy( () -> setupHelper.start()
+				.withAnnotatedEntityTypes( Model.IndexedEntity.class )
+				.setup() )
+				.isInstanceOf( SearchException.class )
+				.satisfies( FailureReportUtils.hasFailureReport()
+						.typeContext( Model.Included.class.getName() )
+						.failure(
+								"An @IndexedEmbedded defines excludePaths filters that were already excluded by the parent filter",
+								"Already excluded paths: [subIncludedStringA].",
+								"Check the filters for typos, or remove them if they are already excluded by a parent and hence are not useful."
```

Comment:
Here too, there's no need for a specific filter. The parent `excludePaths` should be detected as useless.